README.md

```diff
@@ -14,6 +14,37 @@ Maestro is built on learnings from its predecessors (Appium, Espresso, UIAutomat
 - Declarative yet powerful syntax. Define your tests in a `yaml` file.
 - Simple setup. Maestro is a single binary that works anywhere.
 
+## Build and Run Instructions
+Build and Package
+
+To build and package the project, run the following commands:
+
+./gradlew build --stacktrace
+
+./gradlew build
+
+./gradlew jar
+
+The JAR file will be generated in the build/libs directory.
+
+## Run the Application
+
+To execute the application, run:
+
+java -jar build/libs/maestro-0.1.0.jar
```

Comment:
Why this route over the one that's in the maestro script in the root directory?

(Not saying this is wrong - I genuinely don't know which route has what tradeoffs)

Reply:
Because it specifies including libraries and the version to run so that subsequently when there is an update the readme will be updated to which version is in vogue

Reply:
But running the `./maestro` script will build and run the current version of the code and not require the readme update, no?